src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerTopPanel.xaml

```diff
@@ -25,47 +27,136 @@
+                    </MultiDataTrigger>
+                    <MultiDataTrigger>
+                      <MultiDataTrigger.Conditions>
+                        <Condition Binding="{Binding IsSelected, RelativeSource={RelativeSource Self}}" Value="true"/>
+                      </MultiDataTrigger.Conditions>
+                      <Setter Property="Visibility" TargetName="_underline" Value="Visible" />
+                      <Setter Property="Foreground" Value="{DynamicResource {x:Static nuget:Brushes.TabSelectedTextBrushKey }}" />
+                    </MultiDataTrigger>
+                  </ControlTemplate.Triggers>
+                </ControlTemplate>
+              </Setter.Value>
+            </Setter>
+          </Style>
+          <Style TargetType="{x:Type TabControl}">
+            <Setter Property="Padding" Value="2"/>
+            <Setter Property="HorizontalContentAlignment" Value="Center"/>
+            <Setter Property="VerticalContentAlignment" Value="Center"/>
+            <Setter Property="Template" >
+              <Setter.Value>
+                <ControlTemplate TargetType="{x:Type TabControl}">
+                  <Grid x:Name="templateRoot" ClipToBounds="true" SnapsToDevicePixels="true" KeyboardNavigation.TabNavigation="Local">
+                    <Grid.ColumnDefinitions>
+                      <ColumnDefinition x:Name="ColumnDefinition0"/>
+                      <ColumnDefinition x:Name="ColumnDefinition1" Width="0"/>
+                    </Grid.ColumnDefinitions>
+                    <Grid.RowDefinitions>
+                      <RowDefinition x:Name="RowDefinition0" Height="Auto"/>
+                    </Grid.RowDefinitions>
+                    <TabPanel x:Name="headerPanel" Background="Transparent" Grid.Column="0" IsItemsHost="true" Margin="2,2,2,0" Grid.Row="0" KeyboardNavigation.TabIndex="1" Panel.ZIndex="1"/>
+                  </Grid>
+                </ControlTemplate>
+              </Setter.Value>
+            </Setter>
+          </Style>
+        </TabControl.Resources>
+
+        <TabItem Name="tabBrowse" Tag="All" Header="{x:Static nuget:Resources.Label_Browse}" IsSelected="True"/>
+        <TabItem Name="tabInstalled" Tag="Installed">
+          <AutomationProperties.Name>
+            <MultiBinding StringFormat=" {0}{1}">
+              <Binding ElementName="textInstalled" Path="Text" />
+              <Binding ElementName="_warningIcon" Path="ToolTip" />
+            </MultiBinding>
+          </AutomationProperties.Name>
+          <TabItem.Header>
+            <StackPanel Orientation="Horizontal">
+              <TextBlock x:Name="textInstalled" Text="{x:Static nuget:Resources.Label_Installed}"></TextBlock>
+              <imaging:CrispImage
+                Name="_warningIcon"
+                Margin="3,0"
+                Width="13"
+                Height="13"
+                Visibility="Collapsed"
+                HorizontalAlignment="Center"
```

Comment:
does horizontalalignment do anything here?

Reply:
I observed that it can slightly change how the image is drawn, depending on the container's width/height. However, I used a magnifier on our icon, and saw no change with Horizontal/Vertical alignment, so I will remove them.